mimic.py

```diff
@@ -0,0 +1,153 @@
+class Node(object):
+
```

Comment:
No need for a blank line after the class declaration, but *when* you'll add class doc string it will be OK.